tests/std/tests/P0896R4_ranges_alg_remove_copy/test.cpp

```diff
@@ -62,15 +68,36 @@ struct instantiator {
                 assert(result.in == wrapped_input.end());
                 assert(result.out.peek() == output + 3);
                 assert(equal(output, expected));
-                assert(projectionCounter == ranges::size(input));
+                assert(counter == ranges::size(input));
             }
         }
     }
 };
 
+using test::Common, test::Sized;
+
+template <class Category, Sized IsSized, Common IsCommon>
+using test_range =
+    test::range<Category, const P, IsSized, test::CanDifference{derived_from<Category, random_access_iterator_tag>},
+        IsCommon, test::CanCompare{derived_from<Category, forward_iterator_tag> || IsCommon == test::Common::yes},
+        test::ProxyRef::no>;
+
 int main() {
-#ifndef _PREFAST_ // TRANSITION, GH-1030

```

Comment:
are we going to remember to close this or are we going to leave it open until the underlying bug is that made this so bad is fixed? (idk if there's any point in that, it's not like the static analyzer or fe people are tracking our github bugs).

Reply:
There are still many test files (11) that cite this issue, so we don't need to worry about forgetting it forever.

Reply:
I grepped, too. So "yes, we are going to remember to close this."